test/integration/kgw_test.go

```diff
@@ -50,7 +50,7 @@ func TestKGW(t *testing.T) {
 
 			// When user deployed a database
 			// invalid kuneiform
-			specifications.DatabaseDeployInvalidSql1Specification(ctx, t, creatorDriver)
+			//specifications.DatabaseDeployInvalidSql1Specification(ctx, t, creatorDriver)
```

Comment:
No need for this test now?

Reply:
ah forgot I wanted to fix this, will try to include that test